core/src/test/java/kafka/test/junit/ZkClusterInvocationContext.java

```diff
@@ -248,5 +261,17 @@ public void stop() {
                 clusterReference.get().tearDown();
             }
         }
+
+        @Override
+        public void rollingBrokerRestart() {
+            if (started.get()) {
+                for (int i = 0; i < clusterReference.get().brokerCount(); i++) {
+                    clusterReference.get().killBroker(i);
+                }
+                clusterReference.get().restartDeadBrokers(true);
+            } else {
```

Comment:
It would be a bit nicer to move this section to the front, to get rid of the nesting here.